### PR TITLE
Introduce 9c-claim-items-test testnet

### PR DESCRIPTION
**Pull request by moreal**
This pull request introduces the `9c-claim-items-test` testnet. the copy of 9c-internal directory.

### CODE REVIEW

File: 9c-internal/9c-claim-items-test/values.yaml [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+  extraArgs:
+  - --tx-quota-per-signer=1
+
+remoteHeadless:
+  image:
+    repository: planetariumhq/ninechronicles-headless
+    pullPolicy: Always
+
+  hosts:
+  - "9c-internal-rpc-1.nine-chronicles.com"
```
Comment (sky1045) [inline 9c-internal/9c-claim-items-test/values.yaml:89]: same as internal

File: 9c-internal/9c-claim-items-test/values.yaml [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+seed:
+  count: 1
+  useTurnServer: false
+  image:
+    repository: planetariumhq/libplanet-seed
+    pullPolicy: Always # Overrides the image tag whose default is the chart appVersion.
+    tag: "git-22bed8ef41ab09258e6404462ec8e07cd038d089"
+
+  hosts:
+  - "9c-internal-tcp.planetarium.dev"
```
Comment (sky1045) [inline 9c-internal/9c-claim-items-test/values.yaml:49]: same as internal

**Review by sky1045 — CHANGES_REQUESTED** [review]

File: 9c-internal/9c-claim-items-test/values.yaml (hunk 1 above)
Comment (moreal) [reply 9c-internal/9c-claim-items-test/values.yaml:89]: Oops... I'll check one more time. Thanks! 🙏🏻

**Review by sky1045 — APPROVED** [review]
👍

File: 9c-internal/9c-claim-items-test/values.yaml (hunk 1 above)
Comment (moreal) [reply 9c-internal/9c-claim-items-test/values.yaml]: @sky1045 I updated another part too. It seems okay to me now. Could you review this pull request again? 🙏🏻

**Comment by moreal** [comment]
Since the `validate-chart` workflow is broken until #722 merged, I merge this pull request with an approval.

**Pull request merged**